.editorconfig

```diff
@@ -126,3 +126,7 @@ dotnet_diagnostic.CA1062.severity = none
 dotnet_diagnostic.CA1303.severity = none
 # Suppressed[CA1001] as not sure how to fix disposable fields within tests
 dotnet_diagnostic.CA1001.severity = none
+dotnet_diagnostic.CA2227.severity = none # list properties should (not?) be readonly
+dotnet_diagnostic.CA2234.severity = none # call HttpClient.PostAsync(Uri, HttpContent)
+dotnet_diagnostic.CA2007.severity = none # consider calling ConfigureAwaiton awaited task
```

Comment:
I think in the other projects we haven't been ignoring this error and have instead been calling `ConfigureAwait(true)` on the end of async actions.

Reply:
Good call, I've removed this suppression and amended the code as stated.